azure/firewall-policy/variables.tf

```diff
@@ -87,31 +87,31 @@ variable "rule_collection_groups" {
     network_rule_collection = optional(map(object({
       description = optional(string)
       action      = string
-      priority    = string
+      priority    = number
       rule = map(object({
-        protocols             = list(string) #Any, TCP, UDP, ICMP
+        protocols             = list(string, "TCP") #Any, TCP, UDP, ICMP
```

Comment:
needs optional() function if setting default

Reply:
removed for now